packages/wct-mocha/src/stacktrace/formatting.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license
+ * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```

Comment:
2018,  or is this not new code?

Reply:
Actually, this stacktrace directory is all literally imported from https://github.com/googlearchive/stacky...  There was no way to demonstrate that in the diff that I guess.

Reply:
Well.  Except I added typings to everything.